src/plugins/vis_type_script/public/renderer/index.tsx

```diff
@@ -22,8 +23,31 @@ const getSandboxDocument = (script: string) => {
       <head>
         <meta http-equiv="content-security-policy" content="default-src none; script-src 'nonce-${nonce}' ${d3Url}">
         <script src="${d3Url}"></script>
+        <script nonce="${nonce}" type="module">
```

Comment:
`@remote-ui/rpc` isn't available as iife module to be accessed globally. For now I used type=module and mjs as a workaround, but I guess we can't use this without a fallback

Reply:
Curios what browsers and versions our official browser support agreement includes. If we can get away with it, this is a great solution.